addons/account/models/account_invoice.py

```diff
@@ -1045,6 +1045,7 @@ def refund(self, date_invoice=None, date=None, description=None, journal_id=None
                 'in_invoice': ('vendor bill refund')}
             message = _("This %s has been created from: <a href=# data-oe-model=account.invoice data-oe-id=%d>%s</a>") % (invoice_type[invoice.type], invoice.id, invoice.number)
             refund_invoice.message_post(body=message)
+            invoice.refund_invoice_id = refund_invoice
```

Comment:
Okay, but you're using a field that already has another use to do this.

Before this commit, if I created a filter to display only refund invoices, I would look up the invoices that had refund_invoice_id != False. Now this won't work anymore.

But it doesn't look like this creates other problems indeed. The field wasn't used much before, could you just rename the string of refund_invoice_id so that it matches its use?